src/task/script.js

```diff
@@ -291,14 +294,17 @@ const handleDateChange = (event) => {
 function handleStatusChange(event = { target: { value: 'AVAILABLE' } }) {
   const assignee = document.getElementById('assigneeInput');
   const endsOnWrapper = document.getElementById('endsOnWrapper');
+  assignee.style.display = 'none';
+  endsOnWrapper.style.display = 'none';
+  document.getElementById('endsOn').value = '';
+
   if (event.target.value === 'ASSIGNED') {
     setDefaultDates();
     assignee.style.display = '';
     endsOnWrapper.style.display = '';
-  } else {
-    assignee.style.display = 'none';
-    endsOnWrapper.style.display = 'none';
-    document.getElementById('endsOn').value = '';
+    estimatedDays.style.display = 'none';
+  } else if (event.target.value === 'AVAILABLE') {
+    estimatedDays.style.display = '';
```

Comment:
An empty 'display' value? I feel like this should be a defined value here. Unsure what an empty display means. Is it guaranteed to be 'unset' in all browsers?
https://developer.mozilla.org/en-US/docs/Web/CSS/display

Reply:
I didn't take this in consideration. Will definitely set empty string to defined value.